.github/workflows/omero_plugin.yml

```diff
@@ -33,5 +33,5 @@ jobs:
       - name: Set up log level
         run: |
           echo 'ivy.retrieve.log=default' >> etc/local.properties
-      - name: Build and run OMERO tests
-        run: .omero/docker $STAGE
+      #- name: Build and run OMERO tests
+      #  run: .omero/docker $STAGE
```

Comment:
Does the command now fail when the new dependency is introduced? I was this under the impression we had a false negative previously

Reply:
The command does not fail. I removed it since it takes time, running some integrations tests should be done in the CI infra. The flake8 is missing, this needs to be checked.

Reply:
So from your side, is there a value in retaining the workflow at all?

Reply:
flake8 check for python code e.g. test but that could be added to the other one.
I have not looked at it yet

Reply:
I think I am going to remove the GHA in this PR and add the flake8 in a follow-up PR since some Python examples need to be adjusted. 
I don't want to push the flake8 related commits to this PR which about the zarr reader and the build

Reply:
Removed